CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to AET will be documented in this file.
 
 ## Unreleased
 **List of changes that are finished but not yet released in any final version.**
+- [PR-394](https://github.com/Cognifide/aet/pull/394) ([#386](https://github.com/Cognifide/aet/issues/386)) Added scroll modifier
```

Comment:
Mentioning `PR-394` here is sufficient, #386 is linked in the description.

Reply:
Fixed.